app/components/clock/Clock.jsx

```diff
@@ -33,7 +54,15 @@ class Clock extends React.Component {
     render() {
         return (
             <span className="clock">
-              {this.state.dateNow}
+            <span className="hours">{this.state.hours}:</span>
```

Comment:
If in future you'll use these separated spans to make stylings, *double-dot* (`:`) will be styles as well. I doubt you want them to be a part of *hours-styling*. (same true for *minutes* `span`).